packages/website/src/theme/MDXComponents/TryInPlayground.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import styles from './TryInPlayground.module.css';
+
+export function TryInPlayground({
+  eslintrcHash,
+}: {
+  eslintrcHash: string;
+}): React.ReactNode {
+  return (
+    <div>
+      <a
+        href={`/play#eslintrc=${eslintrcHash}`}
+        className={styles.tryInPlaygroundLink}
+      >
+        Try this rule in the playground ↗
+      </a>
+      <br />
```

Comment:
There's no need for a `<br />` or container `<div>` that I can see. I like it not taking up any extra vertical space a bit more.

```suggestion
```